x-pack/plugin/autoscaling/src/main/java/org/elasticsearch/xpack/autoscaling/Autoscaling.java

```diff
@@ -222,9 +221,7 @@ public Collection<AutoscalingDeciderService> deciders() {
         );
     }
 
-    public Set<AutoscalingDeciderService> createDeciderServices(AllocationDeciders deciders) {
-        this.allocationDeciders.set(deciders);
+    protected Set<AutoscalingDeciderService> getDeciderServices() {
```

Comment:
nit: I'd prefer to keep the original name, since this does not just return a field and constructs new objects, i.e., the expectation is that it is called once only (or that a unique set of unique deciders is desired).